lib/management/push.js

```diff
@@ -92,7 +89,9 @@ function startWSManagementClient(pushEndpoint, instanceId, token) {
     };
     const agent = createWSAgent(pushEndpoint, process.env, logger);
 
-    const url = `${pushEndpoint}/${instanceId}/ws`;
+    const url = process.env.CI_ORBIT_PUSH_ENDPOINT ||
```

Comment:
Since the push endpoint is already configurable, this could follow the same pattern as for the API endpoint, instead of having a new CI-related env var.

Reply:
Which endpoint are you speaking about? The one used here (line 93) is not configurable, it is hardocded to 'https://push.api.zenko.io/api/v1/instance' in lib/management/index.js
That being said I don't know if there is other use case than tests to change that, the PUSH endpoint is not supposed to change, is it?

Reply:
There is a separate `PUSH_ENDPOINT` env var that is used and is configurable via the helm chart values.

It is different for dev, staging, production and laptop. It's even planned to have one per-customer for specific needs.